### ISSUE TITLE
Refactor getQuickmatch to fetch JSON

### ISSUE THREAD

**Issue by xmclark** [tranhungt/okcupidjs#26]
The `OKCupid.getQuickmatch` method returns data with HTML (as pointed out by @gnutix in #15). The method should be refactored to return JSON instead, so the API is consistently JSON. 

The fetch url needs the `?okc_api=1` appended. A few other endpoints have this, so an example should be easy to find.

**Comment by tranhungt** [comment]
The `?okc_api=1` has been deprecated a few months back I think. We may need another way to get this data.

**Comment by xmclark** [comment]
Yeah some of the `okc_api` seems to be working still, but probably should not be relied upon.

**Comment by tranhungt** [comment]
Oh cool. As long as it's still working. We can figure it out once they deprecate it completely. Thanks!
